docs/blueprints/linux-setup.md

```diff
@@ -107,28 +126,57 @@ Here is a table summarizing the differences between debian packaging system and
 | Disk footprint	| Smaller	| Larger
 | Application startup time	| Default	| Typically longer
 
-We are thinking to use snap for the following reasons:
-- platform agnostic
-- support for daemons
-- auto-update with failure recovery
-One major drawback is Canonical central's repository.
-
-### Updater
-The updater could either be a service or a cron job.
-
+In our case, the disk footprint comparison does not stand, as 'go' bundles all dependencies.
+For comparison, with a storagenode binary of 85 MB quickly packaged we get:
+- snap: 15 MB
+- deb: 16.8 MB
+
+We are thinking of using native packaging for the following reasons:
+- snap is platform agnostic, but still needs snapd to be installed
+- some linux users are reluctant to use snap
+- covering deb and rpm packaging would make us cover most used distributions
+- with proper packaging, we could directly be included in the distributions
+ 
 ## Implementation
-
-- Implement a service running storagenode binary. (if we do not use snap)
+### Debian package
+- create a storj debian git
+- Implement the debian package skeleton using dh-make. It will create a debian directory that contains all the necessary files to generate the package using dpkg-buildpackage. Commit it to the storj debian git.
+- Modify the `debian/rules` file to embed a `config.yaml` template file and create a `/var/storagenode/bin` directory (where the binaries will be put).
```

Comment:
If it's not too much trouble, we should change this to `/var/lib/storj/bin` or `/var/lib/storj/storagenode/bin` to be compliant with the [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard).